features/provider/active_docs.feature

```diff
@@ -0,0 +1,27 @@
+Feature: ActiveDocs pages
+  As a provider
+  I want to manage my ActiveDocs
+
+  Background:
+    Given a provider is logged in
+      And the provider has 1 active doc
+
+  Scenario: Index renders with the proper elements
+    When I go to the provider active docs page
+     And the table should contain the API
+
+  Scenario: Edit and the redirect after failure render with the proper elements
```

Comment:
I think that can be shorten with Edit with failures

Reply:
> I think that can be shorten to "Edit with failures and retry" something like that.

Good :+1: 

> This scenario does not explain if the api docs is already linked to another service or not

Imo that is not relevant for what wants to be tested here, but anyway, just for you to know, it is not linked because it is inside the provider scope :smile: :wink:

Reply:
Oh the `Background` then should explain it :)

Reply:
Do you mean a Background like `Given the active docs does not have any service linked` ?

Reply:
Sorry I meant in the `Feature` description

Reply:
That makes much more sense to me 👍 😉 💃